tools/make_hgrid/create_gnomonic_cubic_grid.c

```diff
@@ -541,17 +541,17 @@ void direct_transform(double stretch_factor, int i1, int i2, int j1, int j2, dou
 
   for(j=j1; j<=j2; j++) for(i=i1; i<=i2; i++) {
       l = j*nxp+i;
-      if ( fabs(c2m1) > EPSLN7 ) {
-	sin_lat = sin(lat[l]); 
-	lat_t   = asin( (c2m1+c2p1*sin_lat)/(c2p1+c2m1*sin_lat) );
+      if ( fabsl((long double) c2m1) > EPSLN7 ) {
```

Comment:
Will this casting to a long double work if the system/compiler does not support them.  I assume in the case of no support, the long double is the same as a double, but not 100% sure.

This comment also applies to the succeeding uses of fabsl with casting in this source file.

Reply:
Thank you for your review.  I'll correct these areas for no quad precision and push an update.